LICENSE

```diff
@@ -46,6 +46,8 @@ The Chapel implementation is composed of two categories of code:
      gmp              optional multi-precision math library        L-GPL
      hwloc            portable NUMA compute node utilities         new BSD
      jemalloc         alternative memory allocator                 BSD-like
+     libfabric        portable networking layer                    FreeBSD
+                                                                   GPL 2
```

Comment:
I'd drop GPL 2.  As I understand it, when it's dual-licensed, we get to pick the license we're using (does that seem right?), and I believe we're considering our use to be under the BSD.

I also think we might just want to call this "BSD" or "BSD 2-clause" which I think are the more typical names for it.  I suspect if we looked at all the other entries here at that are BSD, we might find that some of them are the same as this one? (e.g., Sphinx is).

Reply:
Sounds fine to me.  Yes, we do get to choose our licensing.  I'll just use 'BSD'.